tests/config.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://cloud.google.com/bigquery/docs/locations
+ALL_BIGQUERY_LOCATIONS = [
```

Comment:
Per issue 331644525, could you move this to `bigframes.constants`, please? In that issue, a `frozenset` would be most desirable for fast lookup operations.

Reply:
Actually, let's have Stephanie do this.